### PR TITLE
Issue 5759: Dev Guide - 1st Code Walk through

### LINKED ISSUE #5759: Documentation : Dev Guide - 1st Code Walk through
**Problem description**
Simplify the existing Dev Guide to create applications against Pravega.

**Problem location**
Documentation.

**Suggestions for an improvement**
Improve it.

### PR DESCRIPTION
**Pull request by shrids**
**Change log description**  
Proposed Dev Guide - 1st Code walkthrough

**Purpose of the change**  
Fixes #5759 

**What the code does**  
Dev Guide - 1st Code walkthrough for a sample Pravega application.

**How to verify it**  
Documentation change.

### CODE REVIEW

File: documentation/src/docs/beginner_dev_guide.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+</p>
+</details>
+
+2. From installation package
+
+<details>
+<summary>Commands to start standalone from installation package</summary>
+<p>
+Download the Pravega release from the [GitHub Releases](https://github.com/pravega/pravega/releases).
```
Comment (SrishT) [inline documentation/src/docs/beginner_dev_guide.md:80]: The hyperlink for the text `GitHub Releases` is not showing up correctly

**Review by SrishT — APPROVED** [review]
LGTM. I was able to walk through the guide step-by-step and create a small sample app successfully.

File: documentation/src/docs/beginner_dev_guide.md (hunk 1 above)
Comment (shrids) [reply documentation/src/docs/beginner_dev_guide.md:81]: fixed.

**Comment by shrids** [comment]
@derekm @RaulGracia I have addressed your review comments as part of PR: https://github.com/pravega/pravega/pull/5745

File: documentation/src/docs/beginner_dev_guide.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega - Creating Your First Application
+
+Learn how to create a Hello World Pravega app. This guide covers:
+
+* Starting Pravega standalone.
```
Comment (RaulGracia) [inline documentation/src/docs/beginner_dev_guide.md:14]: Perhaps, with mkdocs items like these require a blank line in between.

File: documentation/src/docs/beginner_dev_guide.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+* Creating a Pravega Stream.
+* Creating an Event Writer and write events into a Pravega Stream.
+* Create an Event Reader and read events from the Pravega Stream.
+
+
+# 1. Prerequisites
+To complete this guide, you need:
+
+* Less than 15 minutes
```
Comment (RaulGracia) [inline documentation/src/docs/beginner_dev_guide.md:23]: Same here, perhaps a blank line is needed.

File: documentation/src/docs/beginner_dev_guide.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+BUILD SUCCESSFUL in 890ms
+2 actionable tasks: 2 executed
+```
+
+</p>
+</details>
+
+## 4.1 Create a Pravega Stream
+
+Let’s first get to know Pravega’s client APIs by creating a stream with a fixed scaling policy of 1 segment. We’ll need a StreamConfiguration to define this.
```
Comment (RaulGracia) [inline documentation/src/docs/beginner_dev_guide.md:132]: `Let’s`, `We’ll` -> looks like the `'` character is not right. 
In the paragraph below, it looks right in `Pravega let's`.

**Review by RaulGracia — APPROVED** [review]
Looks good, just couple of minor points.

File: documentation/src/docs/beginner_dev_guide.md (hunk 2 above)
Comment (shrids) [reply documentation/src/docs/beginner_dev_guide.md:14]: added it.

File: documentation/src/docs/beginner_dev_guide.md (hunk 3 above)
Comment (shrids) [reply documentation/src/docs/beginner_dev_guide.md:23]: added it

File: documentation/src/docs/beginner_dev_guide.md (hunk 4 above)
Comment (shrids) [reply documentation/src/docs/beginner_dev_guide.md]: fixed it :)

**Comment by derekm** [comment]
Latest changes are visible in the nightly docs:

https://pravega.io/docs/nightly/beginner_dev_guide/

File: documentation/src/docs/beginner_dev_guide.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
```
Comment (RaulGracia) [inline documentation/src/docs/beginner_dev_guide.md:2]: The copyright header has changed, please fix it.

**Review by RaulGracia — CHANGES_REQUESTED** [review]
Holding this one until #5882 is merged. Then, we can proceed adding this content in the new structure of docs.

File: documentation/src/docs/beginner_dev_guide.md (hunk 5 above)
Comment (shrids) [reply documentation/src/docs/beginner_dev_guide.md]: Thanks, fixed it.

**Comment by RaulGracia** [comment]
@shrids can you add an entry for this document as a first element in the `Developing Applications` section of the documentation nav panel?

**Review by RaulGracia — APPROVED** [review]

**Comment by shrids** [comment]
@RaulGracia I have updated mkdocs.yml with this change.

**Pull request merged**